src/types/networkSpecsTypes.ts

```diff
@@ -71,8 +71,10 @@ export function isUnknownNetworkParams(
 		| UnknownNetworkParams
 		| EthereumNetworkParams
 ): networkParams is UnknownNetworkParams {
+	const { protocol, pathId } = networkParams as SubstrateNetworkParams;
 	return (
-		(networkParams as UnknownNetworkParams).protocol ===
-		NetworkProtocols.UNKNOWN
+		(protocol === NetworkProtocols.SUBSTRATE &&
+			pathId !== unknownNetworkPathId) ||
```

Comment:
This matches the same case as above.

Reply:
```suggestion
		(isSubstrateNetworkParams(networkParams) ||
```

Reply:
The function name should be changed to `isSubstrateOrUnknownNetworkParams`, or else it should only be true for unknown.

Reply:
It is still unknown network, but need to change `pathId !== unknownNetworkPathId` to `pathId === unknownNetworkPathId`